wildfly/versions/pom.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+   <modelVersion>4.0.0</modelVersion>
+
+   <parent>
+      <groupId>org.infinispan</groupId>
+      <artifactId>infinispan-parent</artifactId>
+      <version>10.0.0-SNAPSHOT</version>
+      <relativePath>../../pom.xml</relativePath>
+   </parent>
+
+   <groupId>org.infinispan.wildfly</groupId>
+   <artifactId>infinispan-wildfly-versions</artifactId>
```

Comment:
Do we need a versions module any more? Or can we just add directly to `infinispan-wildfly`?

Reply:
I didn't want to introduce too many changes here.
There will also need to be a huge cleanup of the build-configuration/pom.xml to remove a ton of unneeded dependencies.

Reply:
Ok that's reasonable. I'm happy with the changes